packages/cli/src/cmds/preview/commonUtils.ts

```diff
@@ -340,7 +340,16 @@ export function mergeProcessEnv(
   }
   const result = Object.assign({}, process.env);
   for (const key of Object.keys(env)) {
-    result[key] = env[key];
+    if (key === "PATH") {
+      for (const k of Object.keys(result)) {
+        if (k.toUpperCase() === "PATH") {
```

Comment:
Linux and macOS environment variable name are case sensitive, so maybe it is better to only `toUpperCase()` on Windows?

Reply:
`toUpperCase()` is just for match, we respect the original name, see this line
```typescript
result[k] = `${env[key]}${path.delimiter}${result[k]}`;
```

Reply:
But I did not handle the case when there is no `PATH`, `Path`, etc. Let me add it.

Reply:
Worse case is contaminating users environment variable named `path` or `Path` or something like that in ngrok process. As long as ngrok does not read environment variables like `Path` or `path`. It is fine.